Civilian-nization of Research, Complete Remap of Research Wing/Containment

LC4492 opened the pull request:
# About the pull request

This PR focuses on trying to expand the RP opportunities and quality of life of the researchers by taking opportunity of the middle deck added by the multi-Z conversion and by improving their "civilian" aspect through a series of changes.

The entirety of upper medical have been re-mapped, and research received an expansion that involves the addition of new rooms, such as a conference room, and changes for the quality of life of the researchers themselves (Players that like to play as researchers have been consulted on what could be done to improve their gameplay). Containment has been transferred down to the middle deck and is now connected to the upper via a staircase localized in the center of the Research Wing.

Researchers now have access to a much larger array of attires, including new shoes, labcoats, uniforms, ties and etcetera, everything to allow for a much bigger aesthetical freedom, without going against the idea of the role itself. Now researchers fall onto the same flags as the Corporate Liaison, and for this reason will not be receiving service medals according to playtime tier anymore (Playtime ranks still apply).

It has been communicated to me that the use of the BIOSUIT is vital for survival in the Research Laboratories, the reason being that it's the only apparel that grants biological protection, being true given that a labcoat, for example, gives a mere 15 points of protection, with the max being 100 (The BIOSUIT has 100 points of protection). For that reason, I raised the protection of labcoats to 50 points (Half of what a BIOSUIT would give) and made a smaller increase of protection in other items that would contribute (latex gloves give an extra 15 points, sterile masks give an extra 15 points). In the meanwhile, the BIOSUIT has received a slowdown, to make it's use less appealing and to incline researchers to use the equipment that has been modified.

Also fixes issue #11380 by making the grass patches inside the Psychatric Care Unit fake.

# Explain why it's good for the game

Must be, not sure, people will tell.


# Testing Photographs and Procedure
<details>
<summary>SCREENSHOTS OF THE MAP CHANGES</summary>

UPPER DECK MEDICAL
<img width="1348" height="625" alt="image" src="https://github.com/user-attachments/assets/06c5f21a-2b35-4a95-a36e-29c7187da2f7" />

MIDDLE DECK CONTAINMENT
<img width="644" height="490" alt="image" src="https://github.com/user-attachments/assets/daef1ec2-912d-4492-aa7c-cc31392fa175" />


</details>


# Changelog

:cl: Blundir. Forest2001. LCMS1.
add: Adds a second heavy-duty stretcher to containment cells to compensate for the longer travel time.
maptweak: The entirety of upper medical have been re-mapped, and research received an expansion that involves the addition of new rooms, such as a conference room, and changes for the quality of life of the researchers themselves (Players that like to play as researchers have been consulted on what could be done to improve their gameplay). Containment have been transferred down to the middle deck and is now connected to the upper via a staircase localized in the center of the Research Wing.
maptweak: Changes to lower medbay. Ladders have been alligned to the original design pre-multiz, door shutters in the Operation Theatres now work separately again, misceallanous allignment and offsetting work for some of the objects.
code: Raises bio-protection of labcoats from 15 to 50. Made sure latex gloves and sterile masks would have 15 of bio-protection each aswell. Changed the role's entry message a little.
code: Added a sub-variation of the medical vendor (for researchers) that use different name, description and icon.
code: Changed and improved prep vendor loadout for researchers. Added a series of new clothing items, allowing bigger aesthetical freedom. Made small spelling adjusts to other prep vendor loadouts in the file.
code: Changes area code for containment and removes the upperdeck "fakezlevel" tag.
code: Changes tech points' code so now only recovered alien bodies storaged in containment CELLS will count as being storaged. Leaving bodies in the middle of the hall don't count for points anymore.
code: Improves research display code by checking if a cell uses a shutter before showing it's respective control button.
code: Updates old airlock code to not use single letter elements, recodes "airlock locking" procedure in a way that the airlock will first be closed before trying to lock, instead of being locked open.
imageadd: Adds improved versions of containment walls and windows for a more aesthetically-pleasing mapping. 
imageadd: Adds new three-tile airlock for containment, kudos to Blundir for sending me the core sprites.
imageadd: Adds white "generic" airlock, based on the medical one (it's just the same airlock, but the red crosses are gone), intended to replace the older, silver version in locations where it is still used (bathrooms, bunkrooms, etcetera).
imagetweak: Fixes containment cell tiles and reorganizes them inside the file, fixes a good amount of spriting mistakes and codes all possible directions for the turfs, allowing for a much better mapping experience in overall.
fix: Fixes duplicated wheelchairs in the Debilitated's Storage room.
fix: Fixes the bloodbag box in the Field Doctor's preparation spawning without empty bloodbags and removes the bloodbags on the table that, instead of being actual bloodbags, are useless props.
fix: Fixes being able to fill sandbags shipside by replacing the real grass tiles inside the Psychatric Care Unit with proper, fake ones.
spellcheck: Fix spelling on entry message and 'supervisors' for researchers.
spellcheck: Fix spelling on gear vendor item displays.
/:cl:

LC4492 committed:
First push, everything appears fine for now

LC4492 committed:
Second push - fixes to mapping, new goggles, update to stats of lab stuff

LC4492 committed:
Fixes

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Fourth push, merge and vendor fixes

LC4492 committed:
Small little mapping fix

LC4492 committed:
Other small fixes to mapping

LC4492 committed:
Gloves and mask adjustments

LC4492 committed:
Fixup maps in TGM format

5af440814fa3: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
I forgot to merge, small fix

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
extra map fixes

LC4492 committed:
Merge branch 'New-works-for-Research' of https://github.com/LC4492/CM-Space-Station-13 into New-works-for-Research

LC4492 committed:
small finishing fix, the cherry on top

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 commented on code/game/machinery/vending/vendor_types/crew/medical.dm:
```diff
@@ -276,11 +274,14 @@ GLOBAL_LIST_INIT(cm_vending_clothing_researcher, list(
 
-		list("ACCESSORIES (CHOOSE 1)", 0, null, null, null),
-		list("Black Webbing Vest", 0, /obj/item/clothing/accessory/storage/black_vest, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
-		list("Brown Webbing Vest", 0, /obj/item/clothing/accessory/storage/black_vest/brown_vest, MARINE_CAN_BUY_ACCESSORY, VENDOR_ITEM_REGULAR),
```
In my personal experience, Researchers don't need more space than what a satchel can offer, and botany tools don't need to be with a Researcher all the time. Given those tools are also collective, is amazingly strange for Researchers to freely pocket those items, but I will not be considering this aspect. In the end, those webbings appear to be just unnecessary, and will obstruct the civilian look I'm trying to give to the role.

Ultimate-Fluff replied:
Doctors are civilians as well, but they have tons of webbing vests, pouches, and drop pouches. I believe you should reconsider adding webbings back to researchers; they are in the same department as medical, after all.

Ultimate-Fluff commented:
Hey. At least add some medical wardribes and crowbars to the upper floor. I mapped those in a few updates ago and you got rid of them. 

Also, why did you touch the medical maint while working on research?

LC4492 committed:
Update to gear, sneakers and mapping

LC4492 committed:
Fixup maps in TGM format

0e80052ef525: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
small fix to minimap icon

realforest2001 reviewed (dismissed):
Some bugs and errors that need changing, amongst other things. Please reach out to me on discord for further discussion.

LC4492 committed:
Commended fixes, fix to wall dividers, other mapping changes

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Changes to text

LC4492 committed:
Fixup maps in TGM format

9eeb9ab0669d: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

cm13-github commented:
<!-- tgs_test_merge_comment -->
## Test merge deployment history:

<details><summary>Test Merge Deployed @ 11/12/2025 17:11:35 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 7db9a0d356d011fa465decbf5a98bf992f0cbe82
Pull Request: 2a188d5ba1b54b76b98562a63cfbb5a62cb7a986
Server: 0b63bbccb9dd50ddcad896d851b545f5b809d7f5
</details>
<details><summary>Test Merge Updated @ 11/12/2025 17:24:10 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 54b48e4bf5682cf004fc1108391d23b3e8d87ea7
Pull Request: 2a188d5ba1b54b76b98562a63cfbb5a62cb7a986
Server: b53f2c149c7d8941e277ffda7c8d42e37212df2c
</details>
<details><summary>Test Merge Updated @ 11/12/2025 17:34:02 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 35806b74edf02a24a2215b123f758ed3d9fa1e0c
Pull Request: 2a188d5ba1b54b76b98562a63cfbb5a62cb7a986
Server: b49c92c88d3f9d17411aec69221c0b9d29dcfd6f
</details>
<details><summary>Test Merge Removed @ 11/12/2025 19:00:30 +00:00:</summary>

##### Server Instance
CM13 Live
</details>
<details><summary>Test Merge Deployed @ 11/16/2025 18:18:27 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: c2e0825ad9e63d1f826d9537b36e349d3095b3cd
Pull Request: 8d220b530f379da02879e10b9fdb51b68677ea81
Server: 3e49ab7cf4c1d3905824049f078df5f569f30c1a
</details>
<details><summary>Test Merge Updated @ 11/16/2025 18:33:34 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: cba3a14e05ab06fa5f55af955f5535a5ef80c1c8
Pull Request: 8d220b530f379da02879e10b9fdb51b68677ea81
Server: c14287d0e4a7436ad3d812199b450ab6ccdfc78a
</details>
<details><summary>Test Merge Updated @ 11/16/2025 18:40:09 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: cfa5931881b707164f4367c3c7bb5ece8527a617
Pull Request: 8d220b530f379da02879e10b9fdb51b68677ea81
Server: 63d3f66b88695707e73015a21a7e05934800d6d9
</details>
<details><summary>Test Merge Removed @ 11/16/2025 19:04:24 +00:00:</summary>

##### Server Instance
CM13 Live
</details>
<details><summary>Test Merge Deployed @ 11/17/2025 15:37:26 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 1ccb014bf4b2063e1e57b96f1c15cf6643b38029
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 7d5d3f25e10f65708a755efd59e9a6b942320c92
</details>
<details><summary>Test Merge Updated @ 11/17/2025 17:42:40 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 7bd9b6fc3e48adf70c7f09cc84e9b6338c275cc1
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: c57cc4ff9e24facb10ee462d322a2a24c07936e8
</details>
<details><summary>Test Merge Updated @ 11/18/2025 02:15:22 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 9d73f38c83d0f24b29921bd1eb1c23f28670090c
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: e719876a0e851e4275319c2fd60916e11e060f3b
</details>
<details><summary>Test Merge Updated @ 11/18/2025 07:48:08 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 9d73f38c83d0f24b29921bd1eb1c23f28670090c
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: eecaf3c7e40a12b6d393f9b37040ccc63cb3964b
</details>
<details><summary>Test Merge Updated @ 11/18/2025 12:20:46 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 9d73f38c83d0f24b29921bd1eb1c23f28670090c
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 7e6337c6ec978ad322b459bbf01c35ba112c58ca
</details>
<details><summary>Test Merge Updated @ 11/18/2025 15:22:20 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 09a4c4d638deb35fc46a26b2d42face32f49c968
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 4f8c789528f728b83a90fc7f2a9c2242b4803c91
</details>
<details><summary>Test Merge Updated @ 11/18/2025 15:53:11 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 09a4c4d638deb35fc46a26b2d42face32f49c968
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: de9451c684fc91dfa9d16bceb43fbdb28bf1ea41
</details>
<details><summary>Test Merge Updated @ 11/19/2025 01:59:16 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 660fdf086c546d2e741099c5db6afa801571f613
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 8eaa410037fdc7aa54979c4b496eedfed07cb20d
</details>
<details><summary>Test Merge Updated @ 11/19/2025 06:00:50 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: ce933dee533beeb82f5ecabf7adae7c1af1aeba2
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: aaf77d4276adc6dd111b942cb7c0002bf8d12513
</details>
<details><summary>Test Merge Updated @ 11/19/2025 06:31:41 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 9a52f5df1c7c71949383b044b5a2e4632691e059
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: c38b6b10eff08df3565799e8ae05b14cad226668
</details>
<details><summary>Test Merge Updated @ 11/19/2025 15:13:13 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 9a52f5df1c7c71949383b044b5a2e4632691e059
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 27de760436fff6d155368bddae5ce18eb1151ac7
</details>
<details><summary>Test Merge Updated @ 11/19/2025 15:38:30 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 9a52f5df1c7c71949383b044b5a2e4632691e059
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 6f768f8ca30df73ac48dd019be743e5bd561743d
</details>
<details><summary>Test Merge Updated @ 11/20/2025 02:06:29 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 066ceb592003312cbdddfce5475139f4ebb1a85d
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: f497d944cc22bc35b56c878928fa1df9e21b2c8d
</details>
<details><summary>Test Merge Updated @ 11/20/2025 04:46:43 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 066ceb592003312cbdddfce5475139f4ebb1a85d
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 6fb6a8c8a7aaae9e34a99ac66b9b4e29658df2ac
</details>
<details><summary>Test Merge Updated @ 11/20/2025 09:32:03 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 066ceb592003312cbdddfce5475139f4ebb1a85d
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 3d501fa1b7fe2a28ba002042ebfe1af22e061d1c
</details>
<details><summary>Test Merge Updated @ 11/21/2025 04:40:43 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 066ceb592003312cbdddfce5475139f4ebb1a85d
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 68d31cf74043ad87e6d6b4477c935a2f15e496b9
</details>
<details><summary>Test Merge Updated @ 11/21/2025 07:43:30 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: d1af578c173353e3122b64db2005ddf38e781692
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: d6243e10fabcfee5f6770dca1883f413d62def8a
</details>
<details><summary>Test Merge Updated @ 11/21/2025 08:14:23 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: f09289bf17cb774a36e7a625b808a5ea2a4098f8
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 9973964917bc241ee5f46a060d46f61ff7539c85
</details>
<details><summary>Test Merge Updated @ 11/21/2025 08:45:16 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 764d25de1cd35dd157ea55f070f50701ccd0787f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 20d1779416e2693dc791eb3a119992fedabe0aa2
</details>
<details><summary>Test Merge Updated @ 11/21/2025 22:18:52 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 2886fc806c929d8659cab026c0b8f87e52bb03d2
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 1bf2bc5f2c705faa1e743b69c15f81aeef56488e
</details>
<details><summary>Test Merge Updated @ 11/22/2025 01:50:19 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 3204fed2f225956c23f48e902dd23aa6483df9b4
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 2f66bbf61d0918ad2e725152efabf45e8adb5161
</details>
<details><summary>Test Merge Updated @ 11/22/2025 02:51:19 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: c0d8ce01eff2b3e7b1eb1401bef410fa41ad2fbd
</details>
<details><summary>Test Merge Updated @ 11/22/2025 02:59:39 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 38a5883dcfc2cfddd87603bc0eceb8b5a3ead1dc
</details>
<details><summary>Test Merge Updated @ 11/22/2025 03:41:10 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 551b514781e2e565a3a1b7818c19785d8150ca88
</details>
<details><summary>Test Merge Updated @ 11/22/2025 04:38:37 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: a48a175b9170877a067091f2b833fd0ce7adb8f5
</details>
<details><summary>Test Merge Updated @ 11/22/2025 07:08:56 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 551b514781e2e565a3a1b7818c19785d8150ca88
</details>
<details><summary>Test Merge Updated @ 11/22/2025 12:06:35 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: b1badb119b81805f32c7dd991ce6e3802adeb558
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 64c1ab98991ca372c917349d3e1a1c72a52c2043
</details>
<details><summary>Test Merge Updated @ 11/23/2025 02:26:50 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: b5b674e0005218a9641985235a9267b6a7153afd
</details>
<details><summary>Test Merge Updated @ 11/23/2025 03:54:58 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: e95374c4ff313ce5fc83a7f56d426ec5b3dc36d4
</details>
<details><summary>Test Merge Updated @ 11/23/2025 15:06:13 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 2c88c76af73b7da4f5dad654537e1230b9181237
</details>
<details><summary>Test Merge Updated @ 11/23/2025 18:19:28 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 16e98c05d8997d8fd0f824578c9fd1f8a2313b1f
</details>
<details><summary>Test Merge Updated @ 11/23/2025 19:55:48 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: e19696dc7354c4787f30639911e0e099b5aa1e8d
</details>
<details><summary>Test Merge Updated @ 11/23/2025 21:01:06 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 570f08c2fedb6289fe2a9cc4bfa58d27025bd31f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 872bd2355a6eb1eec73ed6846c5d31131f254095
</details>
<details><summary>Test Merge Updated @ 11/24/2025 03:33:13 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 74682d8bb13df3580f5954709ada7a9cf24e7411
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 57f19eabe0b2c500476eab396931b5a245118044
</details>
<details><summary>Test Merge Updated @ 11/24/2025 04:34:14 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: f8d55f95cbbcb99011347db5cae7d40494e5f853
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 9286debd63677010a6bdf4d32769072a811fda6c
</details>
<details><summary>Test Merge Updated @ 11/24/2025 09:06:01 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: ba0406a99c6ba264386839a4c903d35feec16f2f
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 8e3794282ae10da0c903ce455f06da831021cb00
</details>
<details><summary>Test Merge Updated @ 11/24/2025 09:36:55 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 10906b7778a18b44c8f2eec94d9a3749bcfc96d5
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: 02d0badbd910359896c052d0232c49ecc131b712
</details>
<details><summary>Test Merge Updated @ 11/24/2025 13:08:29 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 35bb04f06fe5ee3a66c7cbde2f0b00047f5fefea
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: c86b67f0cad8aff9f5b418f53d9059ffc6339623
</details>
<details><summary>Test Merge Updated @ 11/24/2025 15:30:42 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 35bb04f06fe5ee3a66c7cbde2f0b00047f5fefea
Pull Request: 5aa8620aa89f63a6a9f3a524bdd23d5edc6ddecb
Server: d8bedcf94fd34268bcccb8d1a72662da4067b238
</details>
<details><summary>Test Merge Updated @ 11/24/2025 16:16:59 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 35bb04f06fe5ee3a66c7cbde2f0b00047f5fefea
Pull Request: e38ac98bb436b915c14236d2ecbd6ac8f3b1e471
Server: 180d9e6746ba02f49037d7b95b35aaa3b9b43e55
</details>
<details><summary>Test Merge Updated @ 11/25/2025 02:12:41 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 6791e451bc811407b6daf545a6cf8e3535ee4e4c
Pull Request: e38ac98bb436b915c14236d2ecbd6ac8f3b1e471
Server: 04631a7803fdc162af7f81b7cd3f352782338d49
</details>
<details><summary>Test Merge Removed @ 11/25/2025 15:51:45 +00:00:</summary>

##### Server Instance
CM13 Live
</details>
<details><summary>Test Merge Deployed @ 04/02/2026 22:46:18 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: df7f6784119efc6c17aaff5e6b60b66a6c26f950
Pull Request: 3a47b943937da1a5706fa82e3b755209f4508f9f
Server: 7c00d44e6fd17c52a1479d0260f578f8081e21c9
</details>
<details><summary>Test Merge Updated @ 04/03/2026 00:47:14 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: df7f6784119efc6c17aaff5e6b60b66a6c26f950
Pull Request: 820420c7e74bc9f8ffb45d82727c19b678db0629
Server: e2e40c32f2476f74a0c3cbc84fdca87da4c3a617
</details>
<details><summary>Test Merge Updated @ 04/03/2026 03:48:18 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: df7f6784119efc6c17aaff5e6b60b66a6c26f950
Pull Request: 36d91f9d255e260f8d0c25c226f2589645fc7e73
Server: da909ba3d6c760d73045b66009f83e673724bfc6
</details>
<details><summary>Test Merge Updated @ 04/04/2026 22:54:44 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 616bfe7884f15acfef619d8a1c2bf97a7a2bbc0c
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 9272bf9a25a9d3b0bc580bf28a40d8079a2409b0
</details>
<details><summary>Test Merge Updated @ 04/05/2026 00:25:38 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: 616bfe7884f15acfef619d8a1c2bf97a7a2bbc0c
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 774e1f2f3867e0b1ff1814f09e6318facd7496c1
</details>
<details><summary>Test Merge Updated @ 04/05/2026 06:57:26 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 0b804dcb4fb21f85ebf2ef28e2c5d924ecebaf61
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: ac703d179b1e8892d0cfb941c1d1a3ba387eee85
</details>
<details><summary>Test Merge Updated @ 04/05/2026 20:30:23 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 3439e0b1ed50312409b5b87bf2b911a47fe73729
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: e60787d1f820d390ad6c67822b4003e8b401ba3d
</details>
<details><summary>Test Merge Updated @ 04/06/2026 03:02:11 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 191eea90bd607fcc8f177d7dcfedc945d4aa2253
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 12949eabd2c6227a59fcfd605846fc15ac3078d0
</details>
<details><summary>Test Merge Updated @ 04/06/2026 17:35:42 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 324b6304e27e4d934cebe5fda5de471fd672e32a
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 58648fb4d57524cc87f9110bbec28ba938448c20
</details>
<details><summary>Test Merge Updated @ 04/07/2026 03:08:12 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: be6352e0b9d49f32f1e628c8f16422e6b11a9048
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: df9167fb1b288c3fb718b5daee1f665c135dae51
</details>
<details><summary>Test Merge Updated @ 04/07/2026 23:12:57 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 08c81ad9e894e3a94f89b91e637002ee70bfdd1e
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 45809fa45d5f5dace5b6b3e7084613ae8d926b78
</details>
<details><summary>Test Merge Updated @ 04/08/2026 03:14:17 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: 643a0c9dbc740aa4453a2f48fabd0b2a6d7bf108
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 0a7a60f690a02e10f3f4e214f88708ff3e71e994
</details>
<details><summary>Test Merge Updated @ 04/08/2026 12:16:31 +00:00:</summary>

##### Server Instance
CM13 Live

##### Revision
Origin: d24e8544b7a7bd708de79fa317220258bc21feb4
Pull Request: 2291d22590297e5d2f89a72c620d26a824ea4905
Server: 9b5476cade542f9c879156f73ed36ad552ee2568
</details>
<details><summary>Test Merge Updated @ 04/09/2026 03:17:38 +00:00:</summary>

##### Server Instance

[CM13 Live](https://github.com/cmss13-devs/cmss13/deployments/activity_log?environment=TGS%3A+CM13+Live)

##### Revision
Origin: f0894f0729dcf89f0eeefb60a78b190bb6354c0a
Pull Request: 387441fb859a90dc1d4f09c6d69e77f316ce32bb
Server: dff083e5744155a13df2b43803e8a5750fdc7dc7
</details>
<details><summary>Test Merge Removed @ 04/09/2026 04:52:33 +00:00:</summary>

##### Server Instance
CM13 Live
</details>

LC4492 committed:
Fixes, new containment, more fixes

LC4492 committed:
Fixup maps in TGM format

9eeb9ab0669d: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
fixes, addons

LC4492 committed:
Fixup maps in TGM format

9eeb9ab0669d: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Merge branch 'master' into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

LC4492 committed:
Fixup maps in TGM format

d5e6db0c6b90: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Fixes

LC4492 committed:
Fixup maps in TGM format

d5e6db0c6b90: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Fixes to tables

Detective-Google reviewed (dismissed):
this PR uses some of the areas that are in the hijack whitelist for the upper mid of the ship on the middeck, allowing hijack to hit the middle level (not currently intended); you should either change the areas used or the list. 

<img width="828" height="949" alt="image" src="https://github.com/user-attachments/assets/6566d3be-7055-4b83-80ad-bee482165c42" />

LC4492 committed:
Fixes and addition to mapping

LC4492 committed:
Fixup maps in TGM format

d5e6db0c6b90: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Removal of containment from possible hijack landing

Steelpoint commented on maps/map_files/USS_Almayer/USS_Almayer.dmm:
<img width="410" height="619" alt="button" src="https://github.com/user-attachments/assets/3fc57aa1-742b-4e60-98a3-795d3b8f54b5" />

Swap the order of the buttons so the north cells are opened by the north button and vice versa.

Steelpoint commented on maps/map_files/USS_Almayer/USS_Almayer.dmm:
<img width="502" height="934" alt="er" src="https://github.com/user-attachments/assets/d1612a6f-2eba-4c6c-8440-60b7e1b3a3b0" />

Reorder the placement of the computers to be logically placed nearest each of the containment cells. For example, the south cell computer should be either closer to the south cell or more centralized.

Steelpoint commented on maps/map_files/USS_Almayer/USS_Almayer.dmm:
Go through your map and check every shuttle and blast door ID, there are a few that are missing IDs and won't close/open when requested. For example, none of the Cell 3 doors function remotely via the buttons/computer.

Steelpoint reviewed (changes requested):
See comments.

LC4492 committed:
New containment airlocks, addition of grant to CL, other changes

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Fixes to mapping, further detailing

LC4492 committed:
Fixup maps in TGM format

0b06e2e1d633: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Small fix

LC4492 committed:
Another fix

LC4492 committed:
Fixes

LC4492 committed:
Order fix to code

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Fixup maps in TGM format

208e09e32d6e: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Fixes

LC4492 committed:
Fixes to things I couldn't fix earlier (Yay!)

LC4492 committed:
Merge branch 'master' into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Fixes

LC4492 committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into New-works-for-Research

LC4492 committed:
Small logic fix

tempelers commented:
Bugged. Shutters can't be opened without engineering in the ORs on the first floor.

LC4492 committed:
Fixes to mapping

LC4492 committed:
Fixup maps in TGM format

72d162a0b268: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Small offset fix

LC4492 committed:
Fixup maps in TGM format

72d162a0b268: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Merge branch 'master' into New-works-for-Research

LC4492 committed:
Fixes and addons, poster addition

LC4492 committed:
Fixup maps in TGM format

616bfe7884f1: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

Detective-Google commented:
<img width="395" height="291" alt="image" src="https://github.com/user-attachments/assets/489bdc78-ba6b-489b-a683-785343191e72" />

you messed up a mapmerge at some point and deleted the button for the CL's safe

Detective-Google commented on code/game/objects/items/devices/portable_vendor.dm:
```diff
@@ -316,12 +314,18 @@
+		list("Delta Squad", 10, /obj/item/device/encryptionkey/delta, "white", "Radio Key for USCM Delta Squad."),
+		list("Echo Squad", 10, /obj/item/device/encryptionkey/echo, "white", "Radio Key for USCM Echo Squad."),
+
+		list("RESEARCH EXCHANGE GRANT", 0, null, null, null),
+		list("Research Grant (10 points)", 30, /obj/item/paper/research_notes/grant/cl_special, "white", "A scientific grant to use as a bargain tool while trading with those pesky researchers."),
```
significant balance change that needs to be atomized into it's own PR and not as part of this one, same goes for the other briefcase cost changes in this PR.

LC4492 commented:
> <img alt="image" width="395" height="291" src="https://private-user-images.githubusercontent.com/48196179/575714401-489bdc78-ba6b-489b-a683-785343191e72.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NzU5NDg4NDMsIm5iZiI6MTc3NTk0ODU0MywicGF0aCI6Ii80ODE5NjE3OS81NzU3MTQ0MDEtNDg5YmRjNzgtYmE2Yi00ODliLWE2ODMtNzg1MzQzMTkxZTcyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNjA0MTElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjYwNDExVDIzMDIyM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWE4NzQxN2Q5NjBjMmRmMjQxYmRkZDA5MWI3ZTU3NTk1MTNlNTFkYTc4MzAzM2M5YmZhZmY4MmU1MjM3MzFiYTMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JnJlc3BvbnNlLWNvbnRlbnQtdHlwZT1pbWFnZSUyRnBuZyJ9.iguvxQ-exiHJ2DoPg1kCdqPx1_M7qeflXU2p8d-oS-4">
> you messed up a mapmerge at some point and deleted the button for the CL's safe

The button wasn't deleted, it was moved to the table, unless I mingled the changes I made and reverted it somehow? But should be there.

LC4492 committed:
Reverts changes to CL briefcase

LC4492 committed:
Additional fixes and patches

LC4492 committed:
Fixup maps in TGM format

616bfe7884f1: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Changes body recovery area requirement and small fix to multi-z

LC4492 committed:
Two bugfixes

LC4492 committed:
Addons to mapping

LC4492 committed:
Fixup maps in TGM format

616bfe7884f1: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

MistChristmas commented:
> > 
> > you messed up a mapmerge at some point and deleted the button for the CL's safe
> 
> The button wasn't deleted, it was moved to the table, unless I mingled the changes I made and reverted it somehow? But should be there.



> > 
> > you messed up a mapmerge at some point and deleted the button for the CL's safe
> 
> The button wasn't deleted, it was moved to the table, unless I mingled the changes I made and reverted it somehow? But should be there.


This is why you document all the things you change.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

LC4492 committed:
Merge branch 'master' into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

LC4492 committed:
Fixes to mapping

LC4492 committed:
Merge branch 'New-works-for-Research' of https://github.com/LC4492/CM-Space-Station-13 into New-works-for-Research

LC4492 committed:
Merge branch 'master' into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

LC4492 committed:
Fixup maps in TGM format

b26246cf7710: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Merge branch 'New-works-for-Research' of https://github.com/LC4492/CM-Space-Station-13 into New-works-for-Research

LC4492 committed:
Fixup maps in TGM format

ec4a93d3b3d4: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Fixes fixes and fixes also addons

LC4492 committed:
Fixup maps in TGM format

ec4a93d3b3d4: maps/map_files/USS_Almayer/USS_Almayer.dmm

Automatically commited by: tools\mapmerge2\fixup.py

LC4492 committed:
Fixes to Kutjevo that use containment tiles

Steelpoint commented:
Mapping LGTM

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

LC4492 committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

LC4492 committed:
Improvements to prep vendor loadout

LC4492 committed:
Funny missing comma

LC4492 committed:
Another small fix

cm13-github commented:
This pull request has conflicts, please resolve those before we can evaluate the pull request.

LC4492 committed:
Update colour.dm

LC4492 committed:
Update paper.dm

LC4492 committed:
Update senior_officers.dm

LC4492 committed:
Merge branch 'master' into New-works-for-Research

cm13-github commented:
Conflicts have been resolved. A maintainer will review the pull request shortly.

LC4492 committed:
Merge branch 'master' into New-works-for-Research

MistChristmas commented on code/modules/gear_presets/uscm_medical.dm:
```diff
@@ -181,18 +181,31 @@
 	name = "USCM Researcher"
 
-	access = list(ACCESS_MARINE_MEDBAY, ACCESS_MARINE_RESEARCH, ACCESS_MARINE_CHEMISTRY, ACCESS_MARINE_MORGUE)
+	access = list(
```
None of the ship side roles really should have Colony access stuff.

MistChristmas commented on code/game/objects/items/storage/boxes.dm:
```diff
@@ -113,6 +103,22 @@
 	new /obj/item/clothing/mask/surgical(src)
 
+/obj/item/storage/box/bloodbag
+	name = "empty box of blood bags"
```
Well it's not empty anymore

LC4492 replied:
That's true!

MistChristmas commented on code/game/jobs/job/civilians/support/doctor.dm:
```diff
@@ -11,13 +11,13 @@
 	allow_additional = 1
 	scaled = 1
-	supervisors = "the chief medical officer"
+	supervisors = "the Chief Medical Officer and the Commander"
```
Again I don't think we need much more than the direct supervisor

MistChristmas commented on code/game/jobs/job/civilians/support/cmo.dm:
```diff
@@ -3,11 +3,11 @@
 	total_positions = 1
 	spawn_positions = 1
-	supervisors = "the acting commanding officer"
+	supervisors = "the Commander and the Unites States Colonial Marines High Command"
```
Again I don't think we need much more than the direct supervisor

MistChristmas commented on code/game/jobs/job/civilians/support/cmo.dm:
```diff
@@ -3,11 +3,11 @@
 	flags_startup_parameters = ROLE_ADD_TO_DEFAULT
 	gear_preset = /datum/equipment_preset/uscm_ship/uscm_medical/cmo
-	entry_message_body = "You're a commissioned officer of the USCM. <a href='"+WIKI_PLACEHOLDER+"'>You have authority and knowledge over everything related to Medbay and Research</a>. All personnel within the medical and research bay must answer to you, while you only answer to the XO and CO. You are in charge of medical staff, surgery, chemistry, stimulants and keeping the marines healthy overall  ...
+	entry_message_body = "As the primary representative of the medical knowledge and practice onboard this vessel, your job is to <a href='"+WIKI_PLACEHOLDER+"'>supervise the medical and research teams and ensure they are well behaved and coordinated</a>. Your authority is ample, even though your real knowledge is not necessarily so, and you should have this in mind when supervising your subordinate ...
```
```suggestion
	entry_message_body = "As the primary representative of the medical knowledge and practice onboard this vessel, your job is to <a href='"+WIKI_PLACEHOLDER+"'>supervise the medical and research teams and ensure they are well behaved and coordinated</a>. Your authority is ample, even though your real knowledge is not necessarily so, and you should have this in mind when supervising your subordinates. You answer to the Commander and, if necessity calls you to take over-all command as the acting Commander, High Command. You are also encouraged to fill the gaps where your teams are inefective or understaffed."
```

MistChristmas commented on code/game/machinery/vending/vendor_types/crew/medical.dm:
```diff
@@ -198,86 +199,110 @@ GLOBAL_LIST_INIT(cm_vending_clothing_nurse, list(
 //------------ RESEARCHER ---------------
 GLOBAL_LIST_INIT(cm_vending_clothing_researcher, list(
```
I don't think we need to overload the amount of options they get for multiple uniforms, labcoats, boots, ties and patches. One is fine. And also labcoats says choose 1 but you get 5

MistChristmas commented on code/modules/gear_presets/uscm_medical.dm:
```diff
@@ -207,9 +220,6 @@
-	new_human.equip_to_slot_or_del(new /obj/item/reagent_container/syringe(new_human), WEAR_R_STORE)
 	new_human.equip_to_slot_or_del(new back_item(new_human), WEAR_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/paper/research_notes/unique/tier_one(new_human), WEAR_IN_BACK)
+	new_human.equip_to_slot_or_del(new /obj/item/reagent_container/syringe(new_human), WEAR_IN_BACK)
```
You also might as well kill the syringe they start with.

LC4492 committed:
Update code/game/jobs/job/civilians/support/cmo.dm

Co-authored-by: MistChristmas <TheIrishGamingLeprechaun@gmail.com>

LC4492 committed:
Update boxes.dm

LC4492 committed:
Update cmo.dm

LC4492 committed:
Update doctor.dm

LC4492 committed:
Update uscm_medical.dm

LC4492 committed:
Update medical.dm

LC4492 committed:
Update medical.dm

LC4492 committed:
Merge branch 'master' of https://github.com/cmss13-devs/cmss13 into pr/LC4492/11008

LC4492 committed:
Last fixes

LC4492 committed:
Fixup maps in TGM format

e166b1a2d969: maps/map_files/White_Antre_Research_Facility/White_Antre_Research_Facility.dmm

Automatically commited by: tools\mapmerge2\fixup.py

MistChristmas commented:
Code LGTM

LC4492 committed:
Update nurse.dm
